src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -90,38 +91,58 @@ public void setName(String name) {
     //
     // ---------------------------------------------------------------------------------------------------------
 
+    /** @return a <b>modifiable</b> Set of filters. Can be  null. All changes in this set will be reflected in the CommonInfo */
     public Set<String> getFiltersMaybeNull() {
         return filters;
     }
 
+    /** @return an unmodifiable Set of filters. Can be empty by never null */
     public Set<String> getFilters() {
         return filters == null ? NO_FILTERS : Collections.unmodifiableSet(filters);
     }
 
+    /** @return true if no filter has been defined  <code>(getFiltersMaybeNull()==null)</code> */
```

Comment:
This comment is backwards -- the method returns true if `filters != null`, not if "no filter has been defined"